ci/pipelines/cf-for-k8s-main.yml

```diff
@@ -352,6 +352,128 @@ jobs:
       params:
         remove: ready-pool
 
+- name: validate-cf-for-k8s-gke-external-blobstore
+  serial: true
+  public: true
+  plan:
+    - in_parallel:
+      - get: cf-for-k8s-develop
+        passed:
+          - test-vendir-sync-on-cf-for-k8s
+          - run-unit-tests
+        trigger: true
+      - get: cf-for-k8s-ci
+
+    - put: ready-pool
+      params: {acquire: true}
+
+    - task: install-minio
+      params:
+        GCP_PROJECT_NAME: ((ci_k8s_gcp_project_name))
+        GCP_PROJECT_ZONE: ((ci_k8s_gcp_project_zone))
+        GCP_SERVICE_ACCOUNT_JSON: ((ci_k8s_gcp_service_account_json))
+      file: cf-for-k8s-ci/ci/tasks/install-minio/task.yml
+      input_mapping:
+        pool-lock: ready-pool
+
+    - task: install-cf
+      input_mapping:
+        pool-lock: ready-pool
+        cf-for-k8s: cf-for-k8s-develop
+      params:
+        GCP_SERVICE_ACCOUNT_JSON: ((ci_k8s_gcp_service_account_json))
+        GCP_PROJECT_NAME: ((ci_k8s_gcp_project_name))
+        GCP_PROJECT_ZONE: ((ci_k8s_gcp_project_zone))
+        APP_REGISTRY_HOSTNAME: https://index.docker.io/v1/
+        APP_REGISTRY_REPOSITORY_PREFIX: ((cf_for_k8s_private_dockerhub.username))
+        APP_REGISTRY_USERNAME: ((cf_for_k8s_private_dockerhub.username))
+        APP_REGISTRY_PASSWORD: ((cf_for_k8s_private_dockerhub.password))
+        USE_EXTERNAL_BLOBSTORE: true
+        DOMAIN: k8s-dev.relint.rocks
+      file: cf-for-k8s-ci/ci/tasks/install-cf-on-gke/task.yml
+
+    - task: run-smoke-test
+      file: cf-for-k8s-ci/ci/tasks/run-smoke-tests/task.yml
+      params:
+        SMOKE_TEST_SKIP_SSL: false
+      input_mapping:
+        cf-for-k8s: cf-for-k8s-develop
+
+    - task: push-test-app
+      file: cf-for-k8s-ci/ci/tasks/push-test-app/task.yml
+      params:
+        APP_NAME: lingering-node-app
+        VERIFY_EXISTING_APP: false
+      input_mapping:
+        cf-for-k8s: cf-for-k8s-develop
+
+    - task: run-external-blobstore-validation-test
```

Comment:
Could this be run in parallel with the smoke-tests to save a little time, or does it rely on smoke-tests having run to see blobs in the blobstore? If so, does it make sense for this task to push an app so that it is self-contained?

Reply:
The test now pushes an app and is now self-contained.